src/MICore/MICommandFactory.cs

```diff
@@ -501,6 +501,34 @@ public virtual void DecodeExceptionReceivedProperties(Results miExceptionResult,
 
         #region Helpers
 
+        /// <summary>
+        /// Generates results that represent an emulated MI stopped record.
+        /// </summary>
+        public async Task<Results> GenerateStoppedRecordResults()
+        {
+            Results threadInfo = await this.ThreadInfo();
+            // Get the current thread identifier
+            string currentThreadId = threadInfo.FindString("current-thread-id");
+            Debug.Assert(!String.IsNullOrEmpty(currentThreadId), "Could not determine current thread.");
+            
+            ValueListValue threads = threadInfo.Find<ValueListValue>("threads");
+            Debug.Assert(null != threads, "Could not get thread information from debuggee.");
```

Comment:
You will get a null ref on the next line after the assert. Is that what you want here?